pkg/remote/transmeta/metakey.go

```diff
@@ -61,3 +63,14 @@ const (
 	HeaderConnectionReadyToReset = "crrst"
 	HeaderProcessAtTime          = "K_ProcessAtTime"
 )
+
+// key of acl token
+// You can set up acl token through metainfo.
+// eg:
+//
+//	ctx = metainfo.WithValue(ctx, "gdpr-token", "your token")
+//	ctx, err := tm.MetainfoClientHandler.WriteMeta(ctx, sendMsg)
```

Comment:
`ctx, err := tm.MetainfoClientHandler.WriteMeta(ctx, sendMsg)`
This line of code is confusing, users don't use MetainfoClientHandler directly, remove it.